operating_unit/security/operating_unit_security.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data noupdate="1">
+
+    <record id="group_multi_operating_unit" model="res.groups">
+        <field name="name">Multi Operating Unit</field>
+        <field name="category_id" ref="base.module_category_usability"/>
+    </record>
+
+    <record id="operating_unit_comp_rule" model="ir.rule">
+        <field name="name">Operating Unit Entry</field>
```

Comment:
It is related to creating/editing an operating unit. Only a user that belongs to the company of the OU can access to it.